parcels/examples/tutorial_nemo_curvilinear.ipynb

```diff
@@ -52,9 +52,13 @@
     }
    ],
    "source": [
-    "filenames = {'U': 'NemoCurvilinear_data/U_purely_zonal-ORCA025_grid_U.nc4',\n",
-    "             'V': 'NemoCurvilinear_data/V_purely_zonal-ORCA025_grid_V.nc4',\n",
-    "             'mesh_mask': 'NemoCurvilinear_data/mesh_mask.nc4'}\n",
+    "data_path = 'NemoCurvilinear_data/'\n",
+    "filenames = {'U': {'lon': data_path + 'mesh_mask.nc4',\n",
+    "                   'lat': data_path + 'mesh_mask.nc4',\n",
+    "                   'data': data_path + 'U_purely_zonal-ORCA025_grid_U.nc4'},\n",
+    "             'V': {'lon': data_path + 'mesh_mask.nc4',\n",
+    "                   'lat': data_path + 'mesh_mask.nc4',\n",
+    "                   'data': data_path + 'V_purely_zonal-ORCA025_grid_V.nc4'}}\n",
```

Comment:
Does this new API for `filenames` also need to be documented? Perhaps explain here in the nemo tutorial, and also add in the docstring of FieldSet.from_netcdf etc?